### ISSUE TITLE
Pin wiring for Rasberry Pi A+

### ISSUE THREAD

**Issue by bantin** [ArduCAM/RaspberryPi#4]
I'm looking to use the Arducam with a Rasberry Pi A+ that is currently running bare-metal (no OS). Would this be possible using the Arducam library?

**Comment by ArduCAM** [comment]
@bantin, bare metal programming is not easy, you have to rewrite all the drivers for I2C and SPI interface. Our library is based on WiringPi, which make life easier.

**Comment by bantin** [comment]
Yeah, I've already both an I2C and an SPI interface. Is there anything which depends on having linux installed?
